Dataset_Metrics/dataset_metrics_runner.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import papermill as pm
+import os
+
+class DatasetMetricsRunner(object):
```

Comment:
Minor style detail; in python3 it is not necessary to subclass object. Historically this was done to explicitly subclass old-style classes.

Classes are now defined as
```python
class DatasetMetricsRunner:
     [...]
```